sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -226,6 +227,8 @@ def __enter__(self) -> "OpenTelemetrySpan":
 
     def __exit__(self, exception_type, exception_value, traceback) -> None:
         # Finish the span.
+        if exception_type:
+            self.add_attribute(_ERROR_SPAN_ATTRIBUTE, str(exception_type.__name__))
```

Comment:
we should use the [fully qualified name] - I guess `exception_type.__module__ + '.' + exception_type.__name__` ? 

Created an issue to clarify it on the otel side - https://github.com/open-telemetry/semantic-conventions/issues/855

Reply:
I would use [`__qualname__`](https://peps.python.org/pep-3155/) and not `__name__`

Reply:
Thanks for the feedback. Updated the error.type value to include the `__module__` path, and switched to `__qualname__` instead of `__name__`.